README.md

```diff
@@ -1,38 +1,22 @@
-# IBM Cloud SDK for Power Cloud
-
+# IBM Power-Go-Client
 The power-go-client project provides the Go SDK for IBM® Power Systems™ Virtual Server.
 
-## Prerequisites
-
-- An [IBM Cloud Account](https://cloud.ibm.com/registration)
-- An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account.
-- Go version 1.17 or above.
+## Requirements
+ - An [IBM Cloud Account](https://cloud.ibm.com/registration)
+ - An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account
+ - [Go](https://golang.org/doc/install) v1.17 or above.
+ - [Go Swagger](https://goswagger.io/install.html) v0.19.0
```

Comment:
This is not a requirement to use the sdk.. so we should not put it here.
Also now we are using the latest version of go-swagger which is already mentioned.